zipline/pipeline/filters/filter.py

```diff
@@ -488,3 +489,18 @@ def _compute(self, arrays, dates, assets, mask):
                 asset=self._asset, start_date=dates[0], end_date=dates[-1],
             )
         return out
+
+
+class SmoothingFilter(CustomFilter):
```

Comment:
Here are some ideas:
AllTrueInWindowFilter
AllTrueSmoothingFilter
AllTrueFilter
StrictSmoothingFilter
StrictlyTrueFilter

Any of these work/inspire workable ideas?